ucm2/Intel/avs/avs_da7219/Google-Atlas-1.0-HiFi.conf

```diff
@@ -0,0 +1,67 @@
+SectionVerb {
+        EnableSequence [
+                cdev "hw:avsda7219"
```

Comment:
Remove those lines, they are not required for latest ucm. The device is preset by the default. Those lines are required only if multiple ALSA cards are used for one UCM profile.

Reply:
Ack.